src/scripts/main.js

```diff
@@ -1,3 +1,61 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const table = document.querySelector('.field');
+const addRow = document.querySelector('.append-row');
+const removeRov = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
```

Comment:
```suggestion
const addRowButton = document.querySelector('.append-row');
const removeRowButton = document.querySelector('.remove-row');
const addColumnButton = document.querySelector('.append-column');
const removeColumnButton = document.querySelector('.remove-column');
```
So far, your variables sound like a function, but they are actually just buttons.
It might be confusing